src/PrintFeature.h

```diff
@@ -1,12 +1,14 @@
 #ifndef PRINT_FEATURE
 #define PRINT_FEATURE
 
+#include <cstdint>
+
 namespace cura
 {
 
-enum class PrintFeatureType
+enum class PrintFeatureType: std::uint8_t
 {
-    NoneType, // unused, but libArcus depends on it
+    NoneType, // used to mark unspecified jumps in polygons. libArcus depends on it
```

Comment:
I think a better example would be the extruder switch. When you switch extruder (in place) you don't actually have a travel move to the location of the new nozzle, so the next path is not connected to the previous.

Is that an example of when NoneType should be used?